lib/src/controls/better_player_clickable_widget.dart

```diff
@@ -13,7 +13,7 @@ class BetterPlayerMaterialClickableWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     return Material(
       color: Colors.transparent,
-      child: InkWell(
+      child: GestureDetector(
```

Comment:
Why did you change from InkWell to GestureDetector? InkWell was intentional here, because we show user ripple effect.